examples/hitl/rearrange_v2/rearrange_v2.py

```diff
@@ -391,6 +399,16 @@ def _on_close(self, e: UI.CloseEventData):
             }
         )
 
+    def _on_state_change(self, e: UI.StateChangeEventData):
```

Comment:
@zephirefaith You can use this callback to signal the agent that a state was changed.